### ISSUE TITLE
Poseidon server, Ragnarok client, OpenKore  can run  on Different one computer？

### ISSUE THREAD

**Issue by kkkkiddddd** [OpenKore/openkore#3606]
Poseidon server, Ragnarok client, OpenKore  can run  on Different one computer？

**Comment by alisonrag** [comment]
yes, you can run poseidon and ragnarok in different computer, just need to allow external connection in your router and firewall, also iirc you need to chenge 127.0.0.1 ip to 0.0.0.0

**Comment by kkkkiddddd** [comment]
I have tried to run the same computer to access the Poseidon server, but Poseidon prompts that the connection is successful on different computers, but the Ragnarok is stuck all the time. Is it because my firewall is not set properly?[
![1111](https://user-images.githubusercontent.com/111482855/186069177-6e0a629f-dd8a-4e62-bb09-d2849427fbe4.png)
](url)

**Comment by alisonrag** [comment]
what server?

**Comment by kkkkiddddd** [comment]
cRO

**Comment by alisonrag** [comment]
> cRO

paste your poseidon.txt here

**Comment by Hack3rRat** [comment]
> 在此處粘貼您的poseidon.txt
Hi. alisonrag
I would like to ask if there is a connection or data packet tutorial for twRO now?

**Comment by alisonrag** [comment]
> > 在此處粘貼您的poseidon.txt
> > Hi. alisonrag
> > I would like to ask if there is a connection or data packet tutorial for twRO now?

nope, twRO is using CheatDefender, you will need to find a way to encrypt the data or send data throught the client

**Comment by kkkkiddddd** [comment]
# Ragnarok Server
# Here you'll define the IP Address and the Port where Poseidon
# will keep waiting for your ragnarok online client to connect.
ragnarokserver_ip=0.0.0.0
ragnarokserver_port=6900

# Query Server
# Here you'll define the IP Address and the Port where Poseidon
# will keep waiting for open kore to connect and send the GG/HS queries.
queryserver_ip=0.0.0.0
queryserver_port=24390

# Server Type
# Here you have to specify your current server type in order
# to the poseidon operate properly !
# The available server types for now are : Default, bRO_.* (check servertypes.txt)
# You should modify this if you're having problems with char list.
server_type=cRO_2021-06-25

debug=1

**Comment by kkkkiddddd** [comment]
# Ragnarok Server
# Here you'll define the IP Address and the Port where Poseidon
# will keep waiting for your ragnarok online client to connect.
ragnarokserver_ip=0.0.0.0
ragnarokserver_port=6900

# Query Server
# Here you'll define the IP Address and the Port where Poseidon
# will keep waiting for open kore to connect and send the GG/HS queries.
queryserver_ip=0.0.0.1
queryserver_port=24390

# Server Type
# Here you have to specify your current server type in order
# to the poseidon operate properly !
# The available server types for now are : Default, bRO_.* (check servertypes.txt)
# You should modify this if you're having problems with char list.
server_type=cRO_2021-06-25

debug=1

**Comment by kkkkiddddd** [comment]
ragnarokserver_ip=0.0.0.0
ragnarokserver_port=6900

queryserver_ip=0.0.0.1
queryserver_port=24390

server_type=cRO_2021-06-25

debug=1

**Comment by kkkkiddddd** [comment]
May I ask if my firewall is not set properly because I cannot enter the interface of selecting server while waiting?
#
![QQ图片20220830010424](https://user-images.githubusercontent.com/111482855/187255609-d30ba73e-e799-429b-af28-5e7b5939bdbd.png)

**Comment by alisonrag** [comment]
setup to connect seems ok, the issue should be in packets, the information about cRO client is from 2021-06-05 it has more then 1 year

**Comment by kkkkiddddd** [comment]
It is normal for Poseidon and Ragnarok to be placed on the same computer. You can enter the Poseidon server, but it is monitored by GameGuard

**Comment by kkkkiddddd** [comment]
Excuse me, where can I learn how to extract data packets？

**Comment by kkkkiddddd** [comment]
Is it here to learn how to extract data packets?https://openkore.com/wiki/Connectivity_Guide

**Comment by kkkkiddddd** [comment]
It is normal for Poseidon and Ragnarok to be placed on the same computer. You can enter the Poseidon server, but it is monitored by GameGuard 
![111](https://user-images.githubusercontent.com/111482855/187349673-6ca92d2e-31b2-4699-b567-adc856fc5f4a.png)

**Comment by FrozenCube** [comment]
Can close since it maybe already was provided by our .xe dev

**Comment by CloudMake** [comment]
> 

there is a bug on Poseidon server the server's ip addresss can't be too long.
for example : here you used 192.168.124.124.12 didn't work, then try 192.168.24.2, it will be fine

**Comment by alisonrag** [comment]
> > 
> 
> there is a bug on Poseidon server the server's ip addresss can't be too long. for example : here you used 192.168.124.124.12 didn't work, then try 192.168.24.2, it will be fine

![image](https://user-images.githubusercontent.com/10372732/211026199-47909fc9-4b07-4f2f-b822-f5b037587b3a.png)

your example of ip is wrong, ip only have 4 numbers

**Comment by CloudMake** [comment]
> > > 
> > 
> > 
> > there is a bug on Poseidon server the server's ip addresss can't be too long. for example : here you used 192.168.124.124.12 didn't work, then try 192.168.24.2, it will be fine
> 
> ![image](https://user-images.githubusercontent.com/10372732/211026199-47909fc9-4b07-4f2f-b822-f5b037587b3a.png)
> 
> your example of ip is wrong, ip only have 4 numbers

sorry for the typo, I mean use 192.168.24.2 instead of 192.168.124.12 :)
